src/theme/shared/sizes.ts

```diff
@@ -1,6 +1,5 @@
 const sizes = {
   xs: '1rem',
-  s: '1.5rem',
```

Comment:
After checking with Sergiu about the size variations we found that this value could be replaced with `md` and `lg`